QWC2Components/components/IdentifyViewer.jsx

```diff
@@ -107,7 +107,11 @@ const IdentifyViewer = React.createClass({
     setHighlightedFeatures(feature, resultTree) {
         let features = [];
         if(feature) {
-            features = [feature];
```

Comment:
I think it would be more elegant to just change `setHighlightedFeatures(feature, resultTree)` to `setHighlightedFeatures(features, resultTree)` ( so feature**s** plural) and ensure `setHighlightedFeatures` always receives an array from the caller.